addons/wowl/doc/dropdown_component.md

```diff
@@ -0,0 +1,186 @@
+# Dropdown Component
+
+## Overview
+
+As dropdowns are common in Odoo, we decided to make a generic dropdown component.
+
+It contains all the logic of what you can usually expect a dropdown to behave.
+
+We made the choice to develop this simple component with as few style as possible, therefore it is your responsibility to make yours look awesome 😎!
+
+## Behind the scene
+
+A `<Dropdown/>` component is simply a `<div class="o_dropdown"/>` having a `<button/>` next to an unordered list (`<ul/>`). The button controls if the list is attached to the DOM or not.
+
+A `<DropdownItem/>` is simply a list item (`<li/>`) transporting a payload (which you'll receive back in a custom `dropdown-item-selected` event when the item get clicked).
+
+Illustration of what the final DOM could look like:
+
+```html
+<div class="o_dropdown">
+  <button>Click me to toggle the dropdown menu !</button>
+  <!-- following <ul/> list will or won't appear in the DOM depending on the state controlled by the button -->
+  <ul>
+    <li>Menu Item 1</li>
+    <li>Menu Item 2</li>
+  </ul>
+</div>
+```
+
+## Usage
+
+### Step 1: make it appear on your app
+
+In order to properly use a `<Dropdown/>` component, you need to populate two OWL slots:
+
+<dl>
+  <dt><strong>The <code>default</code> slot</strong></dt>
```

Comment:
not mardown ?

Reply:
Also, I'd move this in the API section

Reply:
I did not find how to do a description list in Markdown :-D